components/AppToolbar.vue

```diff
@@ -1,9 +1,14 @@
+<!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
```

Comment:
I think you added this because eslint started to complain about something in this component. Please avoid using disabling of rules, if its not strictly wanted to leave them out. This rule should just create a warning, no error, so you should be able to deploy just fine.

To us, having a warning here, shows that there's still some work to do - in your case, there's text which should be translated. By having this eslint rule disabled, we have to technical mechanism of that there are things still not final.

Reply:
Ok. Which text should be translated please ?